### PR TITLE
Always show subject in message header when split mode is active

### LINKED ISSUE #1915: message subject no longer shown on message view
### Expected behavior
the message subject should be shown in the message view

### Actual behavior
the message view no longer shows the message subject

### Steps to reproduce
1. open a message
2. try to read the message subject in the message view

### Environment
K-9 Mail version: 5.201

Android version: 7.1.1

Account type (IMAP, POP3, WebDAV/Exchange): IMAP

### PR DESCRIPTION
**Pull request by cketti**
I opted for minimal amount of changes rather than clean code. Hopefully this doesn't live long and we'll get to the UI overhaul soon.

Fixes #1915

### CODE REVIEW

File: k9mail/src/main/java/com/fsck/k9/activity/MessageList.java [hunk 1]
```diff
@@ -1435,6 +1435,8 @@ public void enableActionBarProgress(boolean enable) {
     public void displayMessageSubject(String subject) {
         if (mDisplayMode == DisplayMode.MESSAGE_VIEW) {
             mActionBarSubject.setText(subject);
```
Comment (philipwhiuk) [inline k9mail/src/main/java/com/fsck/k9/activity/MessageList.java]: What is is subject when it's not the email subject. Why are we having to do this crap? What's calling this with an empty string?

Comment (Valodim) [inline k9mail/src/main/java/com/fsck/k9/activity/MessageList.java]: did you go on github again before you had a coffee? :)

Comment (philipwhiuk) [inline k9mail/src/main/java/com/fsck/k9/activity/MessageList.java]: I'm just curious why we are ignoring the passed in `subject` in the else block. It feels wrong.

Comment (cketti) [inline k9mail/src/main/java/com/fsck/k9/activity/MessageList.java]: The call in the else block is only a trampoline to `MessageHeader.showSubjectLine()`. The `MessageHeader` instance already knows about the subject, so it's not passed in as an argument.

**Pull request merged**